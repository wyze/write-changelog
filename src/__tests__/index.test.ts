jest.mock('read-pkg')
jest.mock('execa')
jest.mock('fs')

const date = new Date(2017, 4, 1)

global.Date = jest.fn(() => date)

describe('write-changelog', () => {
  it('works', async () => {
    const fs = require('fs')

    await require('../').default

    expect(fs.writeFileSync.mock.calls).toMatchSnapshot()
  })
})
