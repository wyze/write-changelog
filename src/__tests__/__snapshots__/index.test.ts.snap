// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`write-changelog works 1`] = `
Array [
  Array [
    "changelog.md",
    "
### [v1.0.0](https://github.com/user/repo/releases/tag/v1.0.0) (2017-05-01)

* [[\`c0489425d5\`](https://github.com/user/repo/commit/c0489425d5)] - Initial Commit (Neil Kistner)",
  ],
  Array [
    "readme.md",
    "
### [v1.0.0](https://github.com/user/repo/releases/tag/v1.0.0) (2017-05-01)

* [[\`c0489425d5\`](https://github.com/user/repo/commit/c0489425d5)] - Initial Commit (Neil Kistner)
",
  ],
]
`;
